content/english/Kusto-KC7/activity-6.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Activity 5: Catch the Hacker"
+description: ""
+draft: false
+weight: 16
+---
+
+
+{{< notice note >}}This is a much shorter section, we're just going to explain the scenario and let you go for free hunting time. You can take however much time you'd like to on this portion, but for live sessions we generally give participants 30-40 minutes before moving on.
+
+ **Happy Hunting!**{{< /notice >}}
+
+{{< youtube iXvwu-enXLs >}}
+
+### Introducing the Hackers
+**[Be sure you use the SecurityLogs2 database for this exercise.]**
+
+Now that you’ve completed your initial round of training, you are ready to work your first case in the SOC!
+
+A security researcher tweeted that the domain *“immune.tech”* was being used by hackers. Apparently the hackers are sending this domain inside credential phishing emails.
+
+<img src= "https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/CTF1.png?raw=true" alt= “” width="value" height="value">
+
+{{< alert theme="warning" >}}
+⚠️NOTE! This domain and others encountered in this game are fictional and are not representative of actual malicious activity.{{< /alert >}}
+
+According to OSINT research your colleagues conducted, this domain may be used as part of a phishing campaign with the following stages:
+
+<img src= "https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/CTF2.png?raw=true" alt= “” width="value" height="value">
+
+{{< notice note >}}🎯**Key Point – Open Source Intelligence (OSINT)**: Security researchers and analysts often use free, publicly available data, like Twitter! We call this public data OSINT, and it can be a great way to get investigative leads. Like all public data sources on the internet, you should follow up any OSINT tip with rigorous analysis, rather than blindly trusting the source.{{< /notice >}}
+{{< alert theme="success" >}}
+🤔 **Consider the following questions related to this tip to help you think through the hunt:**
+
+*1.	Which users in our organization were sent emails containing the domain immune.tech?*
```

Comment:
To make this more consumable, perhaps use the {{% showanswer %}} directive to hide these questions in a box?

```suggestion
{{% showanswer Question 1%}}	*Which users in our organization were sent emails containing the domain immune.tech?* {{ %/ showanswer %}}
```

Reply:
The answers are on the scoreboard. I don't want to put them on the page as well that feels too redundant and directs them away from using the scoreboard to check answers. These are just questions to get them thinking.

Reply:
Done for all questions in activity 4 and for the Bonus 1